lib/PGObject/Util/DBAdmin.pm

```diff
@@ -559,26 +559,62 @@ sub server_version {
 }
 
 
-=head2 list_dbs([$dbname])
+=head2 list_dbs([$dbname],[$excludes])
```

Comment:
This implies `my $excludes = shift` with a list of excludes passed as an arrayref. The implementation below takes all remaining arguments as excludes. I like this definition better than the implementation, because the latter does not allow further extension of the API.